Example/Example/RenderModelSelectionViewController.swift

```diff
@@ -74,24 +74,22 @@ class RenderModelSelectionViewController: UIViewController {
 
     private func loadRenderModels() {
         activity.startAnimating()
-        // Checking for the correct license info
-        // Don't forget to enter your configuration string in WannaSDKDefaults.swift
-        if (WannaSDKDefaults.clientConfig == "WSNEAKERS_CLIENT_CONFIG_CHANGE_HERE") {
-            showError(message:"Please change the config string",retry: { [weak self] in
-                self?.loadRenderModels()
-            }, cancel: {
-
-            })
-            return
-        }
         // This is where we create the model storage
         // that will load and set the models to virtual try-on
         do {
             storage = try WsneakersUISDKRenderModelStorage.createStorage(withConfig: WannaSDKDefaults.clientConfig,
                                                                          cacheSize: WannaSDKDefaults.cacheSize)
         } catch {
             activity.stopAnimating()
-            showError(message: error.localizedDescription) { [weak self] in
+            var description = error.localizedDescription
+
+            // Checking for the correct license info
+            // Don't forget to enter your configuration string in WannaSDKDefaults.swift
+            if WannaSDKDefaults.clientConfig == "CHANGE_CLIENT_CONFIG_HERE" {
```

Comment:
I'd add this string to constants in separate file or as a static property so it always be the same here and in `WannaSDKDefaults`

Reply:
I'm afraid that it would be possible to update this new constant instead of `WannaSDKDefaults.clientConfig`

Reply:
<img width="854" alt="image" src="https://user-images.githubusercontent.com/15264480/201098435-33650e71-e956-4616-9015-48318d66a61b.png">

Reply:
they should change this string. not constant

Reply:
in case above it could mislead SDK user